R/teal_lockfile.R

```diff
@@ -92,23 +83,33 @@ teal_lockfile_downloadhandler <- function() {
     },
     content = function(file) {
       teal_lockfile <- "teal_app.lock"
-      iter <- 1
-      while (!file.exists(teal_lockfile) && iter <= 100) {
-        logger::log_trace("lockfile not created yet, retrying...")
-        Sys.sleep(0.25)
-        iter <- iter + 1 # max wait time is 25 seconds
-      }
```

Comment:
This can be removed as `downloadHandler` will be only shown once `lockfile_status()` is run
so just after the `process <- teal_lockfile()` is finished (`process$is_alive() == FALSE`).